org.flatpak.Gourmet.yml

```diff
@@ -0,0 +1,54 @@
+app-id: org.flatpak.Gourmet
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: gourmet
+finish-args:
+  - --share=network
+modules:
+  - name: cpython
+    sources: 
+      - type: archive
+        url:  https://www.python.org/ftp/python/3.8.4/Python-3.8.4.tar.xz
+        sha256:  5f41968a95afe9bc12192d7e6861aab31e80a46c46fa59d3d837def6a4cd4d37
+        
+  - name: intltool
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url:  https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz
+        sha256:  67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd
+
+  - name: dbus-python
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app
+    sources:
+      - type: git
+        url:  https://gitlab.freedesktop.org/dbus/dbus-python
+               
+  - name: PrePipDependencies
+    buildsystem: simple
+    build-options:
+      build-args:
+        - --share=network
+    build-commands:
+      - pip3 install pyenchant sphinx pgi PyGObject gobject
+               
+  - name: PipDependencies
+    buildsystem: simple
+    build-options:
+      build-args:
+        - --share=network
+    build-commands:
+      - pip3 install pygtkspellcheck reportlab lxml keyring dogtail scrape-schema-recipe ebooklib sqlalchemy Pillow gobject beautifulsoup4 pycairo selenium pyglet giofile
+
+  - name: gourmet
+    ensure-writable:
+      - /lib/python2.7/site-packages/easy-install.pth
```

Comment:
`python3.8`?

Reply:
Sorry, I started the flatpak for the original Gourmet, and forgot to remove that line when I switched to this fork for python 3.  It is removed now.